### PR TITLE
Flash Pen for Heads revolutioners

### PR DESCRIPTION
**Pull request by NotSoDana**
## About the PR

Added roundstart item for Headrevs - Flashpen. It's a pen with more powerful lightbulb, which gives rather better stun, slowness and flash.

## Why / Balance

* New item will be logically better than just a security flash for rp.
* No metagaming from Secs, cuz you can insert this pen in ur PDA or don't give it them. Tbh pen a way less suspicious than flash lol.
* Some hardcore for Headrev, flashpen only has 3 charges.
* Men in black reference.

## Technical details
Stun - 2.5s instead of 1.5s;
AoeFlashDuration - 6s instead of 2s;
Range of flash - 5 tiles instead of 7;
Slowness modifier - 0.25 instead of 0.5;

_Optional (not added): autorecharge of flash with 300-600s cooldown_

## Media

https://github.com/space-wizards/space-station-14/assets/75203942/50d8bd22-8e98-44ae-83d0-746087c8c2be

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: Dezzzix
- add: Added roundstart Headrev's Flashpen!

### CODE REVIEW

**Comment by thebadman4662** [comment]
Might want to hide charges info. If you inspect/hold something and see it in decription/ui, then your character knows it, unless sec is supposed to be able to discover it on first arrest which would allow them to check items for tampering?

**Comment by NotSoDana** [comment]
> Might want to hide charges info. If you inspect/hold something and see it in decription/ui, then your character knows it, unless sec is supposed to be able to discover it on first arrest which would allow them to check items for tampering?

I don't have an idea how secs wanna explain why they need to check your pen without metainfo lol. Anyway, I think it'll be better not to hide charge info, cuz of balance(?)

**Comment by beck-thompson** [comment]
> I don't have an idea how secs wanna explain why they need to check your pen without metainfo lol.

You have too much faith that sec will try to justify it (They wont). I like the idea in general though!

**Comment by thebadman4662** [comment]
> > Might want to hide charges info. If you inspect/hold something and see it in decription/ui, then your character knows it, unless sec is supposed to be able to discover it on first arrest which would allow them to check items for tampering?
> 
> I don't have an idea how secs wanna explain why they need to check your pen without metainfo lol. Anyway, I think it'll be better not to hide charge info, cuz of balance(?)

They check pens because they look for signs of tampering, if tampering is so obvious that inspecting or holding item reveals info, its clearly contraband to be taken away. If you are getting arrested in brig and did not do some really trivial crime that you gonna get very light sentence, you probably should expect full search coming.

Now, if you clearly see its a normal pen(at least from seccie POV) and then your start spamming Z button to use "just in case" its flashpen, epen, ebomb(:trollface:) then its easy ahelp and bwoink, sec needs open uplink, knowledge of such contra IC during round and/or antag cooperation to have justification for checking those.

**Comment by SlamBamActionman** [comment]
See closed PRs #22061 and #23451, this appears to be very similar to those.

**Comment by NotSoDana** [comment]
No way 😨

**Comment by thebadman4662** [comment]
To follow up on stealth items. This metashield covers them.

obraz

So there are stealth items that you can't prove are stealth item unless antags explicitly tells you they are and explains how - thieving gloves. You can't test them as its shielded. Not sure if you can still test them with uplink open, as it was not allowed before so you would need admin stance on this, but as metashield stands I suppose you suddenly know how TGs work as soon as open uplink knowledge is up.

Then we have stealth items that are discovered upon inspection or discovery - your proposal of flash pen, hypopen, epen...

Then we have stealth items that leave clues to find and start looking for them like chameleon TGs, though from sec POV they are only chameleon, not thieving.

**Comment by Brandon-Huu** [comment]
> So there are stealth items that you can't prove are stealth item unless antags explicitly tells you they are and explains how - thieving gloves.

That makes NO SENSE.
Don't they leave chameleon fibers on anything they touch?

File: Resources/Prototypes/Entities/Objects/Weapons/security.yml [hunk 1]
```diff
@@ -222,3 +222,26 @@
     - type: GuideHelp
       guides:
       - Security
+
+- type: entity
+  name: pen
+  suffix: Flash
+  parent: Pen # No way, another syndie pen
+  description: A dark ink pen.
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Weapons/security.yml:230]: shouldnt has name and desc since inherited from the parent

**Comment by slarticodefast** [comment]
The only problem I see with this is that the pens reveal you as a head rev. Normal flashes still have uses for combat, scientists (and if I remember correctly there was a cargo flash bounty PR) and are just minor contraband. I guess you only use it for the first three conversions and have to get rid of the evidence after, but this causes new ways of metagaming, too. If sec finds a flash pen, the game mode is instantly known. Maybe make it turn into a normal pen or hide the "0 charges" message once it is used up?

File: Resources/Prototypes/Entities/Objects/Weapons/security.yml (hunk 1 above)
Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Weapons/security.yml:230]: yes they shouldnt copy paste it either

**Comment by deltanedas** [comment]
chef with a flash is still metagamed regardless if they *could* have asked sci / stolen it from sec

**Comment by Ratonero500** [comment]
I think that maybe to avoid metagaming of flash ,it could be done so that each round each player has a very low chance to start with a flash(excepting rev rounds) because if flashes are not illegal then its possible for crewmembers to have them before their shift start.

**Comment by baddiePRO** [comment]
> The only problem I see with this is that the pens reveal you as a head rev.


what if the flash pen was added to traitor uplink too?

**Comment by NotSoDana** [comment]
> > The only problem I see with this is that the pens reveal you as a head rev.
> 
> what if the flash pen was added to traitor uplink too?

I don't think it's good idea, cuz flash pen very useless for traitors lmao. Only for the lulz? Well....

**Comment by ninruB** [comment]
> what if the flash pen was added to traitor uplink too?

It'd be redundant because we already have the flashtern for 2TC at the uplink which serves the exact same niche of being a stealthy antag flash.

**Comment by NotSoDana** [comment]
> I guess you only use it for the first three conversions and have to get rid of the evidence after, but this causes new ways of metagaming, too.

Well-well, i've thinking about adding an auto-recharging of flash 300-600s/1 charge, like in EMAG. It'll make the flash pen less useless, after three conversions, i guess.

**Comment by NotSoDana** [comment]
oh fu..

**Comment by NotSoDana** [comment]
Finally fixed

**Comment by slarticodefast** [comment]
Hey, we had a vote among the maintainers and decided to close this PR. The main reasons mentioned in the discussion were:
- This will cause further metagaming through PDA checking.
- Finding a flash pen will confirm the game mode, while someone having a normal flash in their inventory still has plausible deniability.
- Stealth items currently cause problems rule-wise since they are not marked as contraband, so we don't want to add more of them until a solution is found for this.

Thank you for your contribution.

**Pull request closed without merging**